plugin-server/tests/main/teardown.test.ts

```diff
@@ -22,6 +22,10 @@ const defaultEvent: PluginEvent = {
     properties: { key: 'value' },
 }
 
+const team = {
+    id: 2,
```

Comment:
suggestion: it's a few more lines to declare all properties from the Team interface, but I'd prefer to avoid casting even in tests. If something changes within the pipeline runner and starts using one of the undeclared properties, it might make the troubleshooting process frustrating.

Reply:
Alternatively, if we only use the team ID in the pipeline runner, we should give it a smaller team interface to work with.

Reply:
Done (added the rest of the properties)